index.html

```diff
@@ -12,6 +12,9 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
remove this 3 lines

Reply:
Screenshot show four lines, which of the lines should I remove.

Is it line 12, 13 and 14

Reply:
I believe I have addressed this request??